skbio/util/_misc.py

```diff
@@ -287,6 +298,13 @@ def flatten(items):
     ['a', 'b', 'c', 'd', 1, 2, 3, 4, 5, 'x', 'y', 'foo']
 
     """
+    warn("flatten is deprecated. Please refer to the following links for "
+         "solutions from the standard python library "
```

Comment:
It might help with readability to add a colon and newline to the end of this line, then put a newline between each of the links. Right now everything is space-separated which makes it a little difficult to read.